### PR TITLE
[VM] Address comments from PR #10

### PR DESCRIPTION
**Pull request by pducolin**
What does this PR do?
---------------------

* rename `aptManager` to `PackageManager`
* fix vm resource name
* remove leftovers

Which scenarios this will impact?
-------------------

ec2

Motivation
----------

Post review work based on
 
https://github.com/DataDog/test-infra-definitions/pull/10

Additional Notes
----------------

### CODE REVIEW

File: aws/ec2/ec2/vm.go [hunk 1]
```diff
@@ -12,11 +12,11 @@ import (
...
 
 type VM struct {
-	context    *pulumi.Context
-	runner     *command.Runner
-	aptManager *command.AptManager
+	context *pulumi.Context
+	runner  *command.Runner
 
 	CommonEnvironment *config.CommonEnvironment
+	PackageManager    *command.AptManager
```
Comment (vboulineau) [inline aws/ec2/ec2/vm.go:19]: Should be the `interface` type

Comment (pducolin) [reply aws/ec2/ec2/vm.go:19]: Just package manager or the VM ? I thought we agreed on using a struct as we only have one implementation of VM so far

https://github.com/DataDog/test-infra-definitions/pull/10#discussion_r1035826425

Comment (vboulineau) [reply aws/ec2/ec2/vm.go:19]: The `PackageManager`, the `command.PackageManager` interface already exists.

Comment (pducolin) [reply aws/ec2/ec2/vm.go:19]: I understand now, it should be `*command.PackageManager` type, which is an `interface`

**Review by vboulineau — APPROVED** [review]

**Pull request merged**